pkg/build/api/types.go

```diff
@@ -54,6 +54,12 @@ const (
 	// BuildConfigPausedAnnotation is an annotation that marks a BuildConfig as paused.
 	// New Builds cannot be instantiated from a paused BuildConfig.
 	BuildConfigPausedAnnotation = "openshift.io/build-config.paused"
+	// BuildCompletedAnnotation is an annotation that marks a Build as completed and
+	// will prevent the build controller from processing it further.
+	BuildCompletedAnnotation = "openshift.io/build.completed"
+	// BuildAcceptedAnnotation is an annotation used to update a build that has been
+	// created so it can be seen build controller queue before a resync.
```

Comment:
BuildAcceptedAnnotation is an annotation used to update a build that can now be run based on the RunPolicy(e.g. Serial).  Updating the build with this annotation forces the build to be processed *by the* build controller queue without waiting for a resync.